tools/skel-pack/package.js

```diff
@@ -1,8 +1,13 @@
 Package.describe({
   name: '~name~',
-  summary: ' /* Fill me in! */ ',
-  version: '1.0.0',
-  git: ' /* Fill me in! */ '
+  // Brief, one-line summary of the package.
+  summary: '',
+  version: '0.0.1',
+  // By default, Meteor will default to using README.md for documentation. Use
```

Comment:
How about replacing the comment with:

// By default, Meteor will default to using README.md for documentation.
// To avoid submitting documentation, set this field to null.

I will change the package skeleton, so we can see what it looks like.